src/reducers/util.js

```diff
@@ -76,12 +76,12 @@ export const keyedCounter = (actionType, payloadKey, statePath = 'meta.cache.id'
     return state
   }
 
-export const keyedHasFreshItems = (actionType, bucket) =>
+export const keyedCountFreshItems = (actionType, bucket) =>
```

Comment:
hmm. if we're going to encourage reuse of this, maybe we can just call it `keyedCount` and have the key in the payload be `count`